plugins/system/stats/src/Field/DataField.php

```diff
@@ -52,8 +53,35 @@ protected function getLayoutData()
 
 		$result = Factory::getApplication()->triggerEvent('onGetStatsData', array('stats.field.data'));
 
-		$data['statsData'] = $result ? reset($result) : array();
+		// If the plugin is not published, we need to compile the stats manually
```

Comment:
```suggestion
		// If the plugin is disabled, we need to compile the stats manually
```

plugins are enabled/disabled not published